addons/hr_work_entry/models/hr_version.py

```diff
@@ -46,11 +46,6 @@ def _get_default_work_entry_type_id(self):
         attendance = self.env.ref('hr_work_entry.work_entry_type_attendance', raise_if_not_found=False)
         return attendance.id if attendance else False
 
-    @ormcache()
-    def _get_default_work_entry_type_overtime_id(self):
```

Comment:
As mentioned by @romaincarlier4 , this method should stay in stable (you can add a comment saying that the method is deprecated and will be deleted in 19.1), and then you'll be able to delete it in the master fw-port :slightly_smiling_face: